modules/n1ql/pages/n1ql-language-reference/searchfun.adoc

```diff
@@ -316,27 +316,30 @@ As an alternative, you could limit the number of results and order them using th
 ----
 SELECT META(t1).id
 FROM `travel-sample` AS t1
-WHERE t1.type = "airline" AND SEARCH(t1.country, "United States");
+WHERE t1.type = "hotel" AND SEARCH(t1.description, "amazing");
 ----
 
 .Results
 [source,json]
 ----
 [
   {
-    "id": "airline_2577"
+    "id": "hotel_22096"
   },
   {
-    "id": "airline_2395"
+    "id": "hotel_27741"
   },
   {
-    "id": "airline_2657"
+    "id": "hotel_25243"
   },
-...
+  {
+    "id": "hotel_20422"
+  }
 ]
 ----
 
-If the full text search index being queried has its default mapping disabled and has a custom type mapping defined ("airline" in the above example), the query needs to specify the type explicitly.
+If the full text search index being queried has its default mapping disabled and has a custom type mapping defined ("hotel" in the above example), the query needs to specify the type explicitly.
+For a sample index definition for the above query, refer to xref:fts-demonstration-indexes.adoc#travel-sample-index-hotel-description[Travel sample index definition over documents of type: "hotel"].
```

Comment:
```suggestion
If the full text search index being queried has its default mapping disabled and has a custom type mapping defined, the query needs to specify the type explicitly.
The above query uses the demonstration index xref:fts:fts-demonstration-indexes.adoc#travel-sample-index-hotel-description[travel-sample-index-hotel-description], which has the custom type mapping "hotel".
```

Reply:
Correct xref as it's refering to a different module. Also suggested some minor wording changes.